api/models.py

```diff
@@ -54,8 +54,10 @@ def scheduled_activities(self):
 
 
 class ScheduledActivity(models.Model):
-    date = models.DateTimeField()
+    date = models.CharField(max_length=10, default="yyyy-mm-dd")
     location = models.CharField(max_length=255)
+    latitude = models.DecimalField(max_digits=10, decimal_places=6, default=0)
+    longitude = models.DecimalField(max_digits=10, decimal_places=6, default=0)
```

Comment:
Do we want to remove the defaults after the fact? Otherwise a ScheduledActivity might be created without a proper longitude and latitude

Reply:
I think I needed to add the default as if you migrate then a value like 0 needs to be added for the existing rows that were created before lat and lng started to be added. Otherwise it errors out due to there being no value

Will add a card to the board as a potential bug and check it later but I didn't see anything come up when I was testing it locally